client/keys.go

```diff
@@ -19,7 +19,13 @@ import (
 
 // KeyCommands registers a subtree of commands to interact with
 // local private key storage.
-func KeyCommands(defaultNodeHome string) *cobra.Command {
+//
+// defaultKeyType is the key type to use when no key type is specified.
+// We recommend using "eth_secp256k1" for chains that are focussed on EVM compatibility
+// and sticking with the Cosmos SDK default of "secp256k1" for other chains.
+//
+// TODO: (on follow up PR) adjust Tendermint mentions to CometBFT
+func KeyCommands(defaultNodeHome, defaultKeyType string) *cobra.Command {
```

Comment:
Based on the implementation, I would rename the `defaultKeyType` to something like `defaultToETHKey` type bool. Because if the user set something that is not cosmos nor ethereum it will default to cosmos anyways

Reply:
yooo that's true, I'll adjust it like that 👍